frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -131,6 +139,67 @@ class MeshesView extends React.Component<Props, { currentlyEditedMesh: ?MeshMeta
   };
 
   render() {
+    const moveToIsosurface = (seedPosition: Vector3) => {
+      Store.dispatch(setPositionAction(seedPosition));
+    };
+    const downloadButton = (segmentId: number) => (
+      <Tooltip title="download isosurface">
+        <Icon
+          key="download-button"
+          type="vertical-align-bottom"
+          onClick={() => Store.dispatch(triggerIsosurfaceDownloadAction(segmentId))}
+        />
+      </Tooltip>
+    );
+    const refreshButton = (segmentId: number, isLoading: boolean) => (
+      <Tooltip title="reload isosurface">
+        <Icon
+          key="refresh-button"
+          type={isLoading ? "loading" : "reload"}
+          onClick={() => {
+            Store.dispatch(refreshIsosurfaceAction(segmentId));
+          }}
+        />
+      </Tooltip>
+    );
+    const deleteButton = (segmentId: number) => (
+      <Tooltip title="delete isosurface">
+        <Icon
+          key="delete-button"
+          type="delete"
+          onClick={() => Store.dispatch(removeIsosurfaceAction(segmentId))}
+        />
+      </Tooltip>
+    );
+    const convertHSLAToCSSString = ([h, s, l, a]) =>
+      `hsla(${360 * h}, ${100 * s}%, ${100 * l}%, ${a})`;
+    const convertCellIdToCSS = id =>
+      convertHSLAToCSSString(jsConvertCellIdToHSLA(id, this.props.mappingColors));
+
+    const renderListItem = (isosurface: Object) => {
+      const { segmentId, seedPosition, isLoading } = isosurface;
+      return (
+        <List.Item
+          actions={[
+            downloadButton(segmentId),
+            refreshButton(segmentId, isLoading),
+            deleteButton(segmentId),
```

Comment:
Ah, I see, the vertical bars and the "wrong" alignment is coming from antd's list. Might be hard to fix then :thinking: For the alignment, I don't have an idea, but for the vertical bar, one could try to simply pass an array with length one, like this:

```
<List.Item
          actions={[
            <div>
              {downloadButton(segmentId)}
              {refreshButton(segmentId, isLoading)}
              {deleteButton(segmentId)}
            </div>
...
```

Reply:
Works nicely, thanks for the suggestion!